app/assets/stylesheets/v3/layout_components/responsive-table.scss

```diff
@@ -198,3 +200,80 @@ $use-respond-mixins: false !default;
     margin-bottom: 0;
   }
 }
+
+// This is the new table css as of Nov 2020. Please use this going forward.
+
+.responsive-table {
+  border-style: none;
+  border-collapse: collapse;
+  width: 100%;
+  table-layout: fixed;
+
+  thead,
+  tbody {
+    border: none;
+  }
+
+  th,
+  td {
+    padding: $base-padding;
+    text-align: left;
+    border-right: 1px solid $bg-grey-light;
+
+    &:last-child {
+      border-style: none;
+    }
+  }
+
+  th {
+    background-color: $bg-grey-lighter;
+  }
+
+  tr {
+    background-color: $white;
+  }
+
+  tbody tr:nth-child(even) {
+    background-color: $bg-grey-lightest;
+  }
+
+  @include breakpoint(small only) {
+    thead {
+      clip: rect(0 0 0 0);
+      height: 1px;
+      margin: -1px;
+      overflow: hidden;
+      padding: 0;
+      position: absolute;
+      width: 1px;
+    }
+
+    tr {
+      display: block;
+      padding: $base-padding;
+      font-size: $base-font-size;
+      border-top: 1px solid $bg-grey-light !important;
```

Comment:
why do we need an important here?

Reply:
There was another class with an `!important` overwriting the styles, but I have changed the parent class since I last took a look, so maybe I can get away with not hiving `!important` now.